sys/include/crypto/ciphers.h

```diff
@@ -76,9 +76,6 @@ typedef struct cipher_interface_st {
     /** Blocksize of this cipher */
     uint8_t block_size;
 
-    /** Maximum key size for this cipher */
-    uint8_t max_key_size;
-
     /** the init function */
```

Comment:
It would be good to extend the documentation of the `init` function, explaining that it's its responsibility to check that the `key_size` is valid for the chosen cipher.

Reply:
Done !  I extended the documentation of the init function.

Reply:
Thanks, looks good